src/main/scala/com/softwaremill/mqperf/Receiver.scala

```diff
@@ -27,19 +27,18 @@ object Receiver extends App {
   }
 }
 
-class ReceiverRunnable(
-  mq: Mq,
-  reportResults: ReportResults,
-  receiveMsgBatchSize: Int) extends Runnable {
+class ReceiverRunnable(mq: Mq,
```

Comment:
[minor] Previous formatting looked compatible with default scalariform output. I believe it's also standard formatting returned by scalafmt, so maybe we should keep it? (and add scalariform/scalafmt by the way :)